### ISSUE TITLE
В окне "Рефакторинг" при переименовании для команды "Снять пометки" - сделать более очевидное пояснение

### ISSUE THREAD

**Issue by tormozit** [1C-Company/1c-edt-issues#727]
### Описание ошибки

В окне "Рефакторинг" при переименовании общего модуля не снимаются все пометки командой "Снять пометки"

### Как воспроизвести

Переименовать общий модуль, к которому есть обращения в многих модулях. Откроется окно "Рефакторинг". Нажать кнопку "Снять пометки". Снимутся (надеюсь как у меня) не все пометки.

### Скриншоты

![изображение](https://user-images.githubusercontent.com/11896251/195099666-95d0d2f5-ca52-4477-936f-2aefecf7bf40.png)

### Ожидаемое поведение

Ожидалось снятие всех пометок. Нужно хотя бы опциональная возможность снять все пометки.

### Лог рабочей области

я

### Версия 1С:EDT

Ruby 2022.1.2

### Операционная система

Windows

### Установленные плагины

Нет плагинов

**Comment by adminimusRU** [comment]
Особенно весело, когда поиск ссылок выдал тысячу ложных срабатываний из-за совпадения имени и приходится тщательно просматривать огромный список.

**Comment by marmyshev** [comment]
Картинка и подпись к команде говорит о том, что снимает со всех недостоверных срабатываний только! Там где прямые (точно-определяемые) ссылки - естественно, не снимает. У вас на скрине именно такие - точные ссылки, снимать нечего.

**Comment by marmyshev** [comment]
@adminimusRU 
> Особенно весело, когда поиск ссылок выдал тысячу ложных срабатываний из-за совпадения имени и приходится тщательно просматривать огромный список.

Да, это бич современного стиля написания кода - наверное тут "типизация кода" поможет улучшить точность... не?

Или если есть примеры где код выглядит как однозначно определяемый а при этом ссылка на него показывается как "неоднозначная" - регистрируйте отдельным пожеланием - скорее всего можно будет как-то определить, сделать точнее поиск.

**Comment by tormozit** [comment]
> Картинка и подпись к команде говорит о том, что снимает со всех недостоверных срабатываний только! Там где прямые (точно-определяемые) ссылки - естественно, не снимает. У вас на скрине именно такие - точные ссылки, снимать нечего.

К сожалению на своей картинке я этого не увидел. Возможно в подсказке к кнопке было что то еще написано. 
Но моя главная проблема - убрать ВСЕ пометки. Эффективного способа сейчас вообще нет.

**Comment by adminimusRU** [comment]
@marmyshev 
> Картинка и подпись к команде говорит о том, что снимает со всех недостоверных срабатываний только! Там где прямые (точно-определяемые) ссылки - естественно, не снимает. У вас на скрине именно такие - точные ссылки, снимать нечего.

гадать о назначении кнопки по картинке дело неблагодарное. А вот подсказка прямым текстом говорит о снятии всех пометок, никакого упоминания о сомнительных в ней нет:
![image](https://user-images.githubusercontent.com/28653631/195583006-ac21974c-92f5-492c-a6db-cfafce8d74ec.png)
Если под результатами полнотекстового поиска имелись ввиду именно они, то понять это пользователю EDT без знания внутренних особенностей затруднительно.
Завести отельное пожелание на уточнение подсказки?

**Comment by marmyshev** [comment]
Подсказка - да, непонятная :( 

Иконка - вот прям такая же иконка как выводится для "ложных", 

давайте преквалицифируем это в пожелание
